examples/containers/configure-cbs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eu
+
+echo "Overriding CBS entrypoint as this gives us declarative control"
+
+# Clean up old logs and make sure we have directories we need
+rm -rf /opt/couchbase/var/lib/couchbase/logs/*
+mkdir -p /opt/couchbase/var/lib/couchbase/{config,data,stats,logs}
+
+echo "Run up Couchbase Server"
+/opt/couchbase/bin/couchbase-server -- -kernel global_enable_tracing false -noinput &
```

Comment:
I presume these parameters are from the official Dockerfile? Worth a comment to that end?

Reply:
Possibly, this really should be the entrypoint of the official Dockerfile. There is also a CBSE on hardening that image which covers it but yeah valid point on comment.